code/modules/food/kitchen/smartfridge/medical.dm

```diff
@@ -1,21 +1,46 @@
+/*
+ * Medical
+ */
+ /obj/machinery/smartfridge/medbay
```

Comment:
erroneous space at the start of this line

Reply:
Fixed that but the check is still angry with me